hardhat.config.ts

```diff
@@ -43,7 +43,7 @@ export default {
       forking: {
         url: process.env.MATIC_URL,
         timeout: 12000000,
-        // blockNumber: 27108687,
+        blockNumber: 31480850,
```

Comment:
Don't commit hardhat config if the change is not essential. It could be conflict for other developers.

Reply:
I commit block numbers to my branches because I would like the tests written for the branch to be consistently working even if the state of the chain changes.